README.adoc

```diff
@@ -147,26 +155,28 @@ container needs to be restarted.
 kubernetes.yaml
 [source, yaml, linenums, role='code_column']
 ----
-include::finish/kubernetes.yaml[tags=**;]
+include::finish/kubernetes.yaml[]
 ----
 
 The readiness and liveness probes are configured for the containers running the `system` 
 and `inventory` microservices.
 
-The readiness probes are configured to poll the [hotspot=25 hotspot=66]`/health/ready` endpoint. The readiness probe 
-determines the READY status of the container as seen in the `kubectl get pods` output. The 
-[hotspot=27 hotspot=68]`initialDelaySeconds` field defines how long the probe should wait before it 
+The readiness probes are configured to poll the [hotspot=ready1 hotspot=ready2]`/health/ready` endpoint.
+The readiness probe determines the READY status of the container as seen in the `kubectl get pods` output.
+The [hotspot=delay1 hotspot=delay3]`initialDelaySeconds` field defines how long the probe should wait before it 
 starts to poll so the probe does not start making requests before the server has started. 
-The [hotspot=30 hotspot=71]`failureThreshold` option defines how many times the probe should fail 
-before the state should be changed from ready to not ready. The [hotspot=29 hotspot=70]`timeoutSeconds` 
-option defines how many seconds before the probe times out. The [hotspot=28 hotspot=69]`periodSeconds` 
+The [hotspot=threshold1 hotspot=threshold3]`failureThreshold` option defines how many times the probe should fail 
```

Comment:
The [hotspot=threshold1 hotspot=threshold3]failureThreshold option defines how many times the probe should fail before the state should be changed from ready to not ready.

---->

The [hotspot=threshold1 hotspot=threshold3]failureThreshold option defines how many times the probe fails before the state changes from ready to not ready.